ner_v1/detectors/textual/text/text_detection.py

```diff
@@ -217,25 +218,31 @@ def _text_detection_with_variants(self):
         """
         original_final_list = []
         value_final_list = []
-        normalization = Normalization()
-        self.text_dict = normalization.ngram_data(self.processed_text.lower(), flag_punctuation_removal=False,
-                                                  stem_unigram=False, stem_bigram=False, stem_trigram=False,
-                                                  stop_words_unigram=True, stop_words_bigram=True,
-                                                  stop_words_trigram=True).copy()
+        # normalization = Normalization()
+        # self.text_dict = normalization.ngram_data(self.processed_text.lower(), flag_punctuation_removal=False,
+        #                                           stem_unigram=False, stem_bigram=False, stem_trigram=False,
+        #                                           stop_words_unigram=True, stop_words_bigram=True,
+        #                                           stop_words_trigram=True).copy()
```

Comment:
Delete code instead of commenting, let git does the hard work of preserving the history.
Same for next block